src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1151,15 +1151,23 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected strings to be the same, but they were different.
String lengths are both 14.
Strings differ at index 6.
Expected: 'Mac-1158.local'
But was:  'Mac-1169.local'
at <ScriptBlock>, /Users/runner/runners/2.164.7/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 256
256:             $result[0].Source | Should -BeExactly $hostName
```